test/helper/storage-handler.test.js

```diff
@@ -55,30 +59,30 @@ describe('helpers storage handler', function() {
     });
     it('calls getItem correctly', function() {
       var key = 'foo';
-      var handler = new StorageHandler();
+      var handler = new StorageHandler({ __tryLocalStorageFirst: true });
       expect(handler.getItem(key)).to.be('test');
       expect(getItemStub.firstCall.args).to.be.eql([key]);
     });
     it('calls setItem correctly', function() {
       var key = 'foo';
       var value = 'bar';
-      var handler = new StorageHandler();
+      var handler = new StorageHandler({ __tryLocalStorageFirst: true });
       handler.setItem(key, value, {});
       expect(setItemSpy.firstCall.args).to.be.eql([key, value, {}]);
     });
     it('calls removeItem correctly', function() {
       var key = 'foo';
-      var handler = new StorageHandler();
+      var handler = new StorageHandler({ __tryLocalStorageFirst: true });
       handler.removeItem(key);
       expect(removeItemSpy.firstCall.args).to.be.eql([key]);
     });
   });
   describe('should use cookie storage', function() {
-    it('when localstorage is not available', function() {
+    it('when __tryLocalStorageFirst is true but localSTorage is not available', function() {
```

Comment:
"localStorage"

Reply:
CAN I USE ALL CAPS LIKE THIS LOCALSTORAGE WHAT DO YOU THINK

Reply:
fixed 😬

Reply:
Either all caps or camel, pick one! 😆